templates/page.html

```diff
@@ -175,21 +171,22 @@ <h3>...or install your own</h3>
         <h3>Run your analysis</h3>
         <figure class="figure">
           <div class="command-line">
-            <div class="line comment"># install reana-client</div>
-            <div class="line"><span class="prompt">$</span>mkvirtualenv reana-client -p /usr/bin/python2.7</div>
+            <div class="line comment"># install REANA client</div>
+            <div class="line"><span class="prompt">$</span>mkvirtualenv reana-client</div>
             <div class="line"><span class="prompt">$</span>pip install reana-client</div>
-            <div class="line"><span class="prompt">$</span>reana-client ping</div>
             <div class="line comment"># create new workflow</div>
-            <div class="line"><span class="prompt">$</span>export REANA_WORKON=$(reana-client workflow create)</div>
-            <div class="line comment"># upload runtime code and inputs</div>
-            <div class="line"><span class="prompt">$</span>reana-client code upload ./code/*</div>
-            <div class="line"><span class="prompt">$</span>reana-client inputs upload ./inputs/*</div>
-            <div class="line comment"># start workflow and check progress</div>
-            <div class="line"><span class="prompt">$</span>reana-client workflow start</div>
-            <div class="line"><span class="prompt">$</span>reana-client workflow status</div>
-            <div class="line comment"># download outputs</div>
-            <div class="line"><span class="prompt">$</span>reana-client outputs list</div>
-            <div class="line"><span class="prompt">$</span>reana-client outputs download myplot.png</div>
+            <div class="line"><span class="prompt">$</span>reana-client create -n my-analysis</div>
+            <div class="line"><span class="prompt">$</span>export REANA_WORKON=my-analysis</div>
+            <div class="line comment"># upload input code and data to the workspace</div>
+            <div class="line"><span class="prompt">$</span>reana-client upload ./code ./data</div>
```

Comment:
(1) Remove explicit `./code` and `./data` in the command to have harmony with the download command (=choosing implicit way), or (2) add `results/plot.png` to download command (=choosing explicit way).